sources/tests/Makefile

```diff
@@ -4,6 +4,7 @@
 export $(shell sed 's/=.*//' ../../.env)
 # Override values from the global .env file.
 export $(shell sed 's/=.*//' .env-e2e-tests)
+export ZEEBE_CLIENT_MAX_RETRIES := $(or $(ZEEBE_CLIENT_MAX_RETRIES), 20)
```

Comment:
If there is a fallback here, do we need it also present on the ".env"?

Reply:
Fallback removed and hard coded in docker compose